code/script.js

```diff
@@ -201,11 +206,11 @@ const CHARACTERS = [
 ]
 
 // Global variables
-let secret
-let currentQuestion
-let charactersInPlay
+let secret;
+let currentQuestion;
+let charactersInPlay;
 
-// Draw the game board
+// Draw the game board (invoked on row 255)
```

Comment:
Looks like the board is generated on row 254 and the setSecret function is invoked on 255. Although, I appreciate that you made a comment about when the function will be invoked :)